### PR TITLE
Adds a guitar!

### PR DESCRIPTION
**Pull request by Nienhaus**
This adds a guitar, I might make a video of it in the morning when I wake up.
@JacobLee1 likes how it sounds.
untitled

### CODE REVIEW

**Comment by Nienhaus** [comment]
I forgot inhands. Will add soon.

**Comment by Nienhaus** [comment]
http://youtu.be/g-ClQ0Q06o4

**Comment by Nienhaus** [comment]
Inhands added @Glloyd

**Comment by Glloyd** [comment]
That is cool as fuck!

**Comment by Nienhaus** [comment]
Then merge it and have Vactor reupdate the server :D

**Comment by Glloyd** [comment]
I'm not at home, so I can't really review the code on my phone.

**Comment by scrdest** [comment]
Did you perchance check how big the resources download is after the commit? Because I'm a bit concerned about it bloating, and if it was, we could always ditch the HORRIBLE violin and its files instead.

**Comment by Nienhaus** [comment]
I didn't see the size but the file size for the sound folder is 1.01 MB. I'm not sure if Glloyd wants the violin removed. @scrdest

**Comment by Miauw62** [comment]
You also need a kazoo for the clown :V

File: code/modules/urist/items/guitar.dm [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+/obj/item/device/guitar
+	name = "guitar"
+	desc = "It's made of wood and has bronze strings."
+	icon = 'icons/urist/items/tools.dmi'
+	icon_state = "guitar"
+	item_state = "guitar"
+	urist_only = 1
+	force = 10
+	attack_verb = list("played metal", "made concert", "crashed", "smashed")
```
Comment (Glloyd) [inline code/modules/urist/items/guitar.dm]: Urist McAssistant made concert Honky McHonkerton with the guitar sounds kind of awkward. Same with played metal. Perhaps change made concert to serenaded, and change played metal to "played metal on" or something along those lines.

**Comment by Glloyd** [comment]
It looks like you've added a hit sound for the guitar, but you haven't set the attack sound anywhere.

**Comment by Nienhaus** [comment]
I didn't mean to add the hitsound.

**Comment by Nienhaus** [comment]
@Glloyd

**Comment by Glloyd** [comment]
:+1: For how fucking sexy Fur Elise sounds.

Title2 Is so haunting too.

**Pull request merged**